src/components/CommentList/CommentList.jsx

```diff
@@ -1,3 +1,12 @@
-export const CommentList = () => (
-  <>Put the list here</>
+import { CommentInfo } from '../CommentInfo/CommentInfo';
+import './CommentList.scss'
+
+export const CommentList = ({ comments }) => (
+  <>
+    <div className="CommentList">
+      {comments?.map(comment => (
+        <CommentInfo key={comment.key} comment={comment} />
```

Comment:
What is `comment.key`? Comment doesn't have such property
![image](https://github.com/mate-academy/react_static-list-of-posts-js/assets/96409563/3e52389b-6757-41ad-85af-a09c5eaba4ba)
```suggestion
        <CommentInfo key={comment.id} comment={comment} />
```